### ISSUE TITLE
Duplicant will not shower/will climb ladders slowly while using FastTrack Mod

### ISSUE THREAD

**Issue by Arvenstar** [peterhaneve/ONIMods#281]
## Expected Behavior

Duplicant take showers as normal, climb ladders as normal

Spaced Out is installed
## Current Behavior
Duplicants will either ignore the shower priority, choosing to idle instead. If they do shower, they frequently start/stop. 

## Mod Versions

Version 0.9.15

## Steps to Reproduce

1.There do not seem to be any sensical set of conditions required to recreate this bug. I will attach a short video that shows what happens, but the bug was not occuring when fast track was not installed, nor did it continue after disabling it. 
2.
3.
4.

## Output Log

[Player.log](https://github.com/peterhaneve/ONIMods/files/9842482/Player.log)
Example video -> https://imgur.com/a/ejHlgNf

Basically, dupes bug out when showering, and Ive also noticed the climb up and down ladders extremely slowly, similar to fire poles.

**Comment by peterhaneve** [comment]
All attempts to reproduce this issue have failed. However, some changes have been made in version 0.9.16 which might fix chores being constantly restarted.
